### ISSUE TITLE
User ID when alarm status is changed

### ISSUE THREAD

**Issue by goran-mahovlic** [Supergiovane/node-red-contrib-sia-ultimate#9]
Hi again :)

Is there any way that I can get some info about user that changed status of the alarm?
Like on the mobile app?

I do not see any clue that I could use to identify a user.

**Comment by Supergiovane** [comment]
Hi
the node outputs the full decoded message sent by your alarm system, as a JSON object. You could see in this ibject, if there's the infos you need. If you coudn't find the user, you cannot get it in other ways, sorry.

**Comment by goran-mahovlic** [comment]
> Hi the node outputs the full decoded message sent by your alarm system, as a JSON object. You could see in this ibject, if there's the infos you need. If you coudn't find the user, you cannot get it in other ways, sorry.

Tnx I will investigate more! 

By the way we had some situation so with your system and our DIY sensors we improved situation a little bit.

Someone left alarm disarmed and doors opened and there is no way that we can detect that situation with alarm.

So we created DIY device that is now sending messages if there is no one in space and if alarm is not ON it notify group on telegram ;)

For now it is working flawlessly, but without your system it would not be possible...
